io.openems.edge.evcs.cluster/bnd.bnd

```diff
@@ -0,0 +1,26 @@
+Bundle-Name: OpenEMS Edge EVCS Cluster
+Bundle-Vendor: OpenEMS Association e.V.
+Bundle-License: https://opensource.org/licenses/EPL-2.0
+Bundle-Version: 1.0.0.${tstamp}
+Bundle-Description: \
+	The provider bundle for io.openems.edge.evcs.cluster \
+	\
+Export-Package:  \
+	io.openems.edge.controller.evcs.collection.api;-provide=true
+
+Private-Package: io.openems.edge.evcs.cluster
+
+-includeresource: {readme.md}
+
+-buildpath: \
+	osgi.enroute.base.api;version=2.1,\
+	io.openems.edge.common;version=latest,\
+	io.openems.edge.controller.evcs;version=latest,\
+	io.openems.edge.evcs.api;version=latest,\
+	io.openems.common,\
+	io.openems.edge.core;version=latest
+
+-testpath: \
+	osgi.enroute.junit.wrapper;version=4.12, \
+	osgi.enroute.hamcrest.wrapper;version=1.3
+
```

Comment:
Always use 'bnd.bnd' of other project as example. This format changed (e.g. for testpath')